Misc/NEWS.d/next/Library/2021-04-03-18-03-44.bpo-43723.uBhBZS.rst

```diff
@@ -0,0 +1,6 @@
+The following `threading` methods are now deprecated:
+`threading.currentThread`, `threading.activeCount`,
+`threading.Condition.notifyAll`, `threading.Event.isSet`,
+`threading.Thread.setName`, `threading.thread.getName`,
+`threading.Thread.isDaemon`, and `threading.Thread.setDaemon`. Patch by
+Jelle Zijlstra.
```

Comment:
You should also announce deprecations at: https://docs.python.org/dev/whatsnew/3.10.html#deprecated